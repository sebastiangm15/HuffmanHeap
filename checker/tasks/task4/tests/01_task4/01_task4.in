6
2 SILIM
24 S8S0I
23 S3BH1
24 SGUFK
16 SOFL4
14 SXTCP
3
SGUFK
SOFL4
S3BH1
