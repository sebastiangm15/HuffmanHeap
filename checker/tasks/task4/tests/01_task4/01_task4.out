S3BH1S8S0ISGUFKSILIMSXTCPSOFL4
